hack/generate-latest-csv.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -e
+
+source hack/common.sh
+
+#sha for noobaa-operator:1.1.0
+NOOBAA_SHA="sha256:d6bfdf02d72e763211a89a692f19a8c25bcc26e0d0cd04bda186333bb47894dd"
+
+# sha for rook/ceph:v1.0.0-526.g3ece503
+ROOK_SHA="sha256:addf210104fd4e222b11529f139fb4e12c44707c7b7d46920cdf46e25a3f6a42"
+
+# Current DEV version of the CSV
+export CSV_VERSION=0.0.1
+
+# Current dependency images our DEV CSV are pinned to
+export ROOK_IMAGE=rook/ceph@$ROOK_SHA
```

Comment:
I default to using `podman` over `docker`. And image SHAs don't work well with it. I'd prefer if we switched to using tags when they're available. Still need to figure out a common way to deal with 'latest' tags.